oat-sa/extension-tao-proctoring
# Feature TR-3100 introduce `de-DE` locale

# [TR-3100](https://oat-sa.atlassian.net/browse/TR-3100)

- feat: introduce `de-DE` locale files

https://user-images.githubusercontent.com/2943256/144998258-2b56f421-55a7-419c-8ff5-1ec4bdb3bbef.mov

## How to test
1. Run `php tao/scripts/taoUpdate.php` to apply the translations
2. Publish a Delivery
3. Set `Require Proctoring` option on the Delivery
4. Launch the Delivery as a test taker with a `de-DE` locale
5. Make sure the waiting-window control button labels are translated
